### PR TITLE
Minor doc chanches

### PR DESCRIPTION
**Pull request by akshat998**
Minor changes to the slurm-readme.md file & the all.ctrl file 

A better description of the `collection_folder` variable is provided

### CODE REVIEW

File: README-slurm.md [hunk 1]
```diff
@@ -140,12 +140,14 @@ Once submitted, the jobs will be visible in the Slurm queue (`squeue`)
 
 #### Monitor Progress
 
-At present, the only way to monitor progress of jobs will be to check status through the Slurm scheduler commands. This will be extended in the future to more natively track progress towards completion.
+The progress of the jobs can be monitored using the Slurm scheduler commands. 
+For debugging, the slurm errors are contained within `/VFVS/workflow/workunits`.
```
Comment (mjkoop) [inline README-slurm.md]: ../workflow/workunits (from tools path)

File: README-slurm.md [hunk 2]
```diff
@@ -140,12 +140,14 @@ Once submitted, the jobs will be visible in the Slurm queue (`squeue`)
...
 
 
 ## Viewing Results
 
 The results of the job will be placed in the location specified within the `all.ctrl`.
+By default, this is saved in: `/VFVS/output-files/{workunit-number}`.
```
Comment (mjkoop) [inline README-slurm.md]: /VFVS is not necessarily

File: tools/templates/all.ctrl [hunk 3]
```diff
@@ -144,13 +144,15 @@ object_store_data_collection_prefix=
...
-# Path to where the collection file are stored
+# Path to where the collection file (processed ligands/docking requirement files) 
+# are stored
 #  * This is used when job_storage_mode=sharedfs or
 #    when the uploader helper script is being used
 #
 # Slash at the end is not required (optional)
 # Either pathname is required w.r.t. the folder tools/
 #     or absolute path (e.g. /home/vfuser/collections)
+# We suggest setting this to: ../input-files/ligand-library
```
Comment (mjkoop) [inline tools/templates/all.ctrl]: I'd recommend against it be in ../input-files. That directory gets tar/gzipped.

File: README-slurm.md [hunk 4]
```diff
@@ -69,16 +69,16 @@ Slurm-specific Configuration:
...
 
 
 The ligands to be processed should be included in the file within `tools/templates/todo.all`. This file can be automatically generated from the VirtualFlow website.
 
 
 ### Data for Virtual Screening
 
-The location of the collection files to be used in the screening should be located in `collection_folder` (defined in `all.ctrl`).
-
+The location of the collection files (containing the list of processed ligands) to be used in the screening should be located in `collection_folder` (defined in `all.ctrl`). Typically, this is set to the input folder (`../input-files/ligand-library`).
```
Comment (mjkoop) [inline README-slurm.md]: I'd recommend not in input-files

File: README-slurm.md [hunk 5]
```diff
@@ -69,16 +69,16 @@ Slurm-specific Configuration:
...
+Apart from the input ligands, all other inputs used for docking (such as the receptor file and docking config file) are located as subdirectories within the `collection_folder`.
```
Comment (mjkoop) [inline README-slurm.md]: Those files are in `docking_scenario_basefolder`

File: tools/templates/all.ctrl [hunk 6]
```diff
@@ -144,13 +144,15 @@ object_store_data_collection_prefix=
 ****** Shared Filesystem Settings
 
 collection_folder=/home/ec2-user/collections
-# Path to where the collection file are stored
+# Path to where the collection file (processed ligands/docking requirement files)
```
Comment (mjkoop) [inline tools/templates/all.ctrl]: perhaps 'ready-to-dock ligands'

**Review by mjkoop — CHANGES_REQUESTED** [review]

File: README-slurm.md (hunk 1 above)
Comment (akshat998) [reply README-slurm.md]: Great point

File: README-slurm.md (hunk 5 above)
Comment (akshat998) [reply README-slurm.md]: AHH!

**Pull request closed without merging**